src/auth/auth.ts

```diff
@@ -38,9 +38,9 @@ export async function getSession(skipPrompt = false): Promise<vscode.Authenticat
   }
 
   signInPrompted = true;
-  const signInAction = "Sign in to GitHub";
+  const signInAction = "Sign into GitHub";
```

Comment:
This and the one below aren't correct. See the "Sign in" text on the normal site. "in to" isn't contracted in these cases
<img width="422" alt="image" src="https://github.com/github/vscode-github-actions/assets/1297909/d62b4bcf-dbbc-460c-8001-2a0475bc6200">

Reply:
In this case, if I go to Github.com in general without being logged in we are consistently using "sign in to":

![image](https://github.com/github/vscode-github-actions/assets/7976517/0450c489-dd8a-4fa7-9110-7ea8e801824c)